modules/silicon/scripts/build/container/jupyterlab/cloudbuild.yaml

```diff
@@ -0,0 +1,38 @@
+timeout: 3600s
+steps:
+- name: 'gcr.io/cloud-builders/git'
+  args: ['clone', '-b', '2022.02.01_02.19.58', 'https://github.com/The-OpenROAD-Project/OpenLane']
```

Comment:
should use https://cloud.google.com/build/docs/configuring-builds/substitute-variable-values instead of hardcoding git tag